core/bindings.rs

```diff
@@ -460,6 +464,54 @@ fn eval_context(
   rv.set(to_v8(tc_scope, output).unwrap());
 }
 
+/// This binding should be used if there's a custom console implementation
+/// available. Using it will make sure that proper stack frames are displayed
+/// in the inspector console.
+///
+/// Each method on console object should be bound to this function, eg:
+/// ```ignore
+/// function wrapConsole(consoleFromDeno, consoleFromV8) {
+///   const callConsole = core.callConsole;
+///
+///   for (const key of Object.keys(consoleFromV8)) {
+///     if (consoleFromDeno.hasOwnProperty(key)) {
+///       consoleFromDeno[key] = callConsole.bind(
+///         consoleFromDeno,
+///         consoleFromV8[key],
+///         consoleFromDeno[key],
+///       );
+///     }
+///   }
+/// }
+/// ```
+///
+/// Inspired by:
+/// https://github.com/nodejs/node/blob/1317252dfe8824fd9cfee125d2aaa94004db2f3b/src/inspector_js_api.cc#L194-L222
+fn call_console(
+  scope: &mut v8::HandleScope,
+  args: v8::FunctionCallbackArguments,
+  _rv: v8::ReturnValue,
+) {
+  assert!(args.length() >= 2);
```

Comment:
This code will cause a Rust panic if callConsole is invoked with no args, or invalid args.
- `Deno.core.callConsole()`
- `Deno.core.callConsole(1)`
- `Deno.core.callConsole(1, 2)`

Reply:
Indeed, this function is not really meant to be called manually by users.